taichi/python/export_lang.cpp

```diff
@@ -326,6 +326,7 @@ void export_lang(py::module &m) {
   m.def("layout", layout);
 
   m.def("value_cast", static_cast<Expr (*)(const Expr &expr, DataType)>(cast));
+  m.def("bits_cast", static_cast<Expr (*)(const Expr &expr, DataType)>(bit_cast));
```

Comment:
Why are there both `bits_cast` and `bit_cast`?

Reply:
value_cast -> cast -> cast_value
bits_cast -> bit_cast -> cast_bits
Let's rename `value_cast` then.